ajo/AJOCampaigns.md

```diff
@@ -0,0 +1,86 @@
+# Summary of personalization content
+
+There are two AJO Campaigns that impact the page.
+
+## Campaign 1
+
+[//]: # "https://experience.adobe.com/#/@aemonacpprodcampaign/sname:prod/journey-optimizer/campaigns/summary/43053ce3-6f3a-4ff0-aaa2-dbe71ec6f231"
+
+Campaign 1 is a web campaign with an experiment, created using the Visual Experience Composer (VEC). Experiment has two treatments with each one changing the header text and adding an alert beneath it.
+
+| Treatment                    | Visual treatment                                                       |
+| ---------------------------- | ---------------------------------------------------------------------- |
+| Treatment A                  | <img src="../.assets/activity-1-exp-A.png" alt="drawing" width="800"/> |
+| Treatment B                  | <img src="../.assets/activity-1-exp-B.png" alt="drawing" width="800"/> |
+| Default (no personalization) | <img src="../.assets/activity-1-exp-0.png" alt="drawing" width="800"/> |
+
+## Campaign 2
+
+[//]: # "https://experience.adobe.com/#/@aemonacpprodcampaign/sname:prod/journey-optimizer/campaigns/summary/ebfb3a5e-71ec-45e2-9e2a-866df9c539cb"
+
+Campaign 2 is a code based experience campaign with an experiment, created using the code editor. Experiment has two treatments with each one consisting of JSON values to update the hero image and button text/actions on the page.
```

Comment:
Campaign 2 is a code based experience campaign with an experiment created using the code editor. The experiment has two treatments with each one consisting of JSON values to update the hero image and button text/actions on the page.